examples/graphlearn_for_pytorch/README.md

```diff
@@ -0,0 +1,69 @@
+# Using GraphLearn-for-PyTorch(GLT) for Distributed Acceleration for PyG
+
+**[GraphLearn-for-PyTorch(GLT)](https://github.com/alibaba/graphlearn-for-pytorch)** is a graph learning library for PyTorch that makes
+distributed GNN training and inference easy and efficient. It leverages the
+power of GPUs to accelerate graph sampling and utilizes UVA to reduce the
+conversion and copying of features of vertices and edges. For large-scale graphs,
+it supports distributed training on multiple GPUs or multiple machines through
+fast distributed sampling and feature lookup. Additionally, it provides flexible
+deployment for distributed training to meet different requirements.
+
+Most of the APIs of GLT are compatible with PyG,
+so you only need to modify a few lines of PyG's code to get the
+acceleration of the program. For GLT-specific APIs,
+they are also compatible with PyTorch and there are complete documentations
+and usage examples available.
+
+
+## Installation
+
+### Requirements
+- cuda>=11.4
```

Comment:
CUDA version greater than 11.4 is not required anymore after GLT removes cudaFreeAsync.

Reply:
deleted.